tutorials/R/download-ais-data/download-NEON-AIS-data.Rmd

```diff
@@ -0,0 +1,307 @@
+* The expanded data tables
+* Dates matching the other data products you've downloaded
+
+1. What is the size of the downloaded data? 
+2. Without downloading all the data, how can you tell what the difference in size
+between the "expanded" and "basic" packages is? 
+</div>
+
+```{r challenge-code-extent-crs, echo=FALSE}
+# download data of interest - Nitrate in Suface Water
+nitr<-loadByProduct(dpID="DP1.20033.001", site="BLDE", 
+											 startdate="2019-10", enddate="2019-11", 
+											 package="basic", 
+											 token = Sys.getenv("NEON_TOKEN"),
+											 check.size = F)
+
+# #1. 73.3 KiB 
+# #2. You can change check.size to True (T), and compare "basic" vs "expaneded"
+# package types. The basic package is 66.2 KiB. 
+
+```
+
+## Files Associated with Downloads
+
+The data we've downloaded comes as an object that is a named list of objects. 
+To work with each of them, select them from the list using the `$` operator. 
+
+```{r loadBy-list}
+# view all components of the list
+names(waq)
+
+# View the dataFrame
+View(waq$waq_instantaneous)
+
+```
+
+We can see that there are four objects in the downloaded nitrate data. One 
+dataframe of data (`waq_instantaneous`) and three metadata files. 
+
+If you'd like you can use the `$` operator to assign an object from an item in 
+the list. If you prefer to extract each table from the list and work with it as an 
+independent objects, which we will do, you can use the `list2env()` function. 
+
+``` {r unlist}
+
+# assign the dataFrame in the list as an object
+#wqInst <- waq$waq_instantaneous
+
+# unlist the vari
+list2env(waq, .GlobalEnv)
+
+```
+
+So what exactly are these four files and why would you want to use them? 
+
+* **data file(s)**: There will always be one or more dataframes that include the 
+primary data of the data product you downloaded. Multiple dataframes are available 
+when there are related datatables for a single data product.
+* **readme_xxxxx**: The readme file, with the corresponding 5 digits from the data
+product number, provides you with important information relevant to the data 
+product and the specific instance of downloading the data.
+* **sensor_postions_xxxxx**: this file contains information about the coordinates 
+of each sensor, relative to a reference location. 
+* **variables_xxxxx**: this file contains all the variables found in the associated
+data table(s). This includes full definitions, units, and other important 
+information. 
+
+## Data from Different Sensor Locations (HOR)
+
+NEON collected the same type of data from sensors in different locations. These 
+data are delivered together but you will frequently want to plot the data seperately 
+or only include data from one sensor in your analysis. NEON uses the 
+`horizontalPosition` variable in the data tables to describes which data a sensor 
+is collected from. The `horizontalPosition` is always three digit number for AIS 
+data. 
+
```

Comment:
I would make this a little more general - something like 
NEON uses the horizontalPosition and verticalPosition variables in the data table to describe which sensor the data are collected from. For the water quality data product, verticalPosition is always 100, indicating in-water measurements. The horizontalPosition is three digits and indicates position in the water body, as follows:

Reply:
Complete. Also linked to the NEON Data Product Numbering Convention doc for more info.